app/models/concerns/appeal_notification_report_concern.rb

```diff
@@ -49,9 +49,9 @@ def upload_notification_report!
   # Returns: The document name
   def notification_document_name
     if is_a?(Appeal)
-      "notification-report_#{uuid}_#{Time.now.utc.strftime('%Y%m%d%k%M%S')}"
+      "notification-report_#{uuid}"
     elsif is_a?(LegacyAppeal)
-      "notification-report_#{vacols_id}_#{Time.now.utc.strftime('%Y%m%d%k%M%S')}"
+      "notification-report_#{vacols_id}"
     end
   end
```

Comment:
```suggestion
  def notification_document_name
    "notification-report_#{external_id}"
  end
```

Using the `external_id` method that is in both `Appeal` and `LegacyAppeal` can make this a one-liner.